src/v2/Apps/Artist/Routes/AuctionResults/ArtistAuctionResults.tsx

```diff
@@ -66,6 +66,19 @@ const AuctionResultsContainer: React.FC<AuctionResultsProps> = ({
   const { pageInfo } = artist.auctionResultsConnection ?? {}
   const { hasNextPage, endCursor } = pageInfo ?? {}
   const artistName = artist.name
+  const queryString = require("query-string")
+
+  // scroll to auction results if param flag is present
+  useEffect(() => {
+    const queryParams = queryString.parse(location.search)
```

Comment:
We can use `useRouter` here to get the query string (here we get the filters the same way https://github.com/artsy/force/blob/6700454cc5fd4ad090a2e92f6523293f2b15e21c/src/v2/Apps/Artist/Routes/AuctionResults/ArtistAuctionResults.tsx#L291).

```
  const { match } = useRouter()
  const { scrollToAuctionResults } = paramsToCamelCase(match?.location.query)
```